src/main/java/teammates/common/datatransfer/attributes/StudentProfileAttributes.java

```diff
@@ -242,8 +245,13 @@ public StudentProfileAttributes build() {
             return profileAttributes;
         }
 
-        private boolean isGenderValid(String gender) {
-            return "male".equals(gender) || "female".equals(gender) || "other".equals(gender);
+    }
+
+    public Gender gendervalidity(String gender) {
```

Comment:
Still not a right place. Why not put this under the `Gender` enum instead?
Also, there has to be a better name than `gendervalidity`.